From 7718dc09f0217115b886a35b80cf38927f1b747e Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Sun, 3 Feb 2019 18:24:38 +0100
Subject: [PATCH 01/11] Revert "sepolicy: recovery: Allow (re)mounting system"

This reverts commit f1ad32105599a0b71702f840b2deeb6849f1ae80.

Change-Id: I00668ef3358d8179357cceadc6fc551ed307c642

# Conflicts:
#	common/private/recovery.te
---
 common/private/recovery.te | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/common/private/recovery.te b/common/private/recovery.te
index 35f1ad4..fe4d63c 100644
--- a/common/private/recovery.te
+++ b/common/private/recovery.te
@@ -28,7 +28,4 @@ set_prop(recovery, system_prop)
 # Switch to backuptool
 allow recovery self:process setexec;
 domain_trans(recovery, otapreopt_chroot_exec, backuptool)
-
-# Some addons require this for A/B installation compatibility
-allow recovery system_file:dir mounton;
 ')
-- 
2.17.1

